services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_api/__init__.py

```diff
@@ -1,5 +1,5 @@
 from ._core import (
-    are_all_services_present,
+    all_dynamic_sidecar_stack_services_present,
```

Comment:
;) not sure I get better what is exactly meant here...
all_services_present
vs
all_dynamic_sidecar_stack_services_present

Without looking in the code, now I'm wondering you mean the actual service (as seen in the osparc GUI) or the services + sidecar + proxy?
I guess it is the latter right? you could define it as a ```pod``` ?

Reply:
Below convention was adapted and function was renamed